paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/TestParameters.kt

```diff
@@ -34,14 +35,16 @@ internal data class TestParameters(
         fun create(
             paymentMethodCode: String,
             requiresBrowser: Boolean = true,
-            authorizationAction: AuthorizeAction = AuthorizeAction.AuthorizePayment(requiresBrowser),
-            playgroundSettingsBlock: (PlaygroundSettings) -> Unit = {}
+            authorizationAction: AuthorizeAction? = AuthorizeAction.AuthorizePayment(requiresBrowser),
+            executeInNightlyRun: Boolean = false,
```

Comment:
I think the introduction of this `executeInNightlyRun` flag means that we now have two sources of truth for which tests are run in browserstack. One is the browserstack script [here](https://github.com/stripe/stripe-android/blob/c351d4adda26782202e3844f9bbbe556e80bdd76/scripts/browserstack.py#L574) and the other is this flag. What do you think about defining which tests to run in browserstack.py instead? That would also make it easier to interpret the results in browserstack, because there wouldn't be so many tests that show up as "skipped" (https://app-automate.browserstack.com/dashboard/v2/builds/b0771330a837e50eddfda75f30e54ea0c1c3d090)

Reply:
Hmm, I consider the kotlin code to be the source of truth. I would probably propose doing the opposite. In my mind we should treat the kotlin code as the source of truth, and the python script only has code like that for determining which tests go in which shard.

Reply:
Chatted offline. I'm going to follow up with a PR that moves all the filtering logic to kotlin code.